Makefile

```diff
@@ -39,6 +39,7 @@ EXEEXT := $(shell $(PYTHON) -c \
 NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
+NPM ?= $(PWD)/deps/npm/bin/npm-cli.js
```

Comment:
I'd probably drop `$(PWD)` because that breaks if the path contains spaces (unless you quote its uses below.)